tests/test_persisted_task_properties.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for validating persisted task properties for different data types."""
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+
+
+class TestPersistedTaskPropertyDataTypes(object):
+    """Contains a collection of pytest tests.
+
+    This validates the property getter,setter and deleter methods for
+    persisted task properties for different data types.
+    """
+
+    @pytest.fixture(scope="class")
+    def voltage_tester_task(self):
+        """Gets the voltage tester task from the persisted tasks in the system."""
+        system = nidaqmx.system.System.local()
+        voltage_tester_task = system.tasks["VoltageTesterTask"]
+        return voltage_tester_task
```

Comment:
Please add a `persisted_task` fixture similar to `persisted_scale` in https://github.com/ni/nidaqmx-python/pull/236, plus the suggested skip logic. Then we can use it in other tests that need persisted tasks, by calling `task = persisted_task.load()`.

Reply:
Added a fixture for generating persisted tasks as suggested